templates/staff/report_detail.html

```diff
@@ -0,0 +1,93 @@
+{% extends "staff/base.html" %}
+
+{% load humanize %}
+
+{% block metatitle %}{{ report.title }}: Staff Area | OpenSAFELY Jobs{% endblock metatitle %}
+
+{% block breadcrumbs %}
+<nav class="breadcrumb-container breadcrumb--danger" aria-label="breadcrumb">
+  <div class="container">
+    <ol class="breadcrumb rounded-0 mb-0 px-0">
+      <li class="breadcrumb-item">
+        <a href="{% url 'staff:index' %}">Staff area</a>
+      </li>
+      <li class="breadcrumb-item">
+        <a href="{% url 'staff:report-list' %}">Reports</a>
+      </li>
+      <li class="breadcrumb-item active" aria-current="page">
+        {{ report.title }}
+      </li>
+    </ol>
+  </div>
+</nav>
+{% endblock breadcrumbs %}
+
+{% block jumbotron %}
+<div class="jumbotron jumbotron-fluid jumbotron--danger pt-md-2">
+  <div class="container">
+    <h1 class="display-4">{{ report.title }}</h1>
+
+    <ul class="list-unstyled lead">
+      <li>
+        <strong>Created by:</strong>
+        <a href="{{ report.created_by.get_staff_url }}">{{ report.created_by.name }}</a>
+      </li>
+
+      <li>
+        <strong>Created at:</strong> {{ report.created_at }}
+      </li>
+    </ul>
+
+    <div class="d-flex">
+      <a class="btn btn-primary mr-1" href="{{ report.release_file.get_absolute_url }}">View on ReleaseFile</a>
```

Comment:
> View on ReleaseFile

This sounds a bit clunky. Maybe "View released output"?

Reply:
Hands up who did `s/Site/ReleaseFile` 😂